### ISSUE TITLE
Implement hash to group element

### ISSUE THREAD

**Issue by isislovecruft** [isislovecruft/ristretto-donna#2]
I've not implemented hash-to-group yet. https://ristretto.group/formulas/elligator.html

**Comment by Warchant** [comment]
Can libsodium be used?
https://download.libsodium.org/doc/advanced/point-arithmetic/ristretto#hash-to-point-elligator

**Comment by isislovecruft** [comment]
@Warchant No, libsodium cannot be used, as that would defeat the purpose of implementing this on top of ed25519-donna.

**Comment by isislovecruft** [comment]
I've started this work in my `feature/ristretto-from-uniform-bytes` branch, which has our flavour of the elligator2 encoding and a `ristretto_from_uniform_bytes()` wrapper API on top of it.

**Comment by calvin2021y** [comment]
hi @isislovecruft

what is the status of ` feature/ristretto-from-uniform-bytes` branch?
